doc/builders/special/darwin-builder.section.md

```diff
@@ -61,3 +61,74 @@ builders-use-substitutes = true
 ```ShellSession
 $ sudo launchctl kickstart -k system/org.nixos.nix-daemon
 ```
+
+## Example flake usage
+
+```
+{
+  inputs = {
+    nixpkgs.url = "path:/Users/geraint.ballinger/Documents/personal/nixpkgs";
+    nixpkgs-unstable.url = github:NixOS/nixpkgs/nixpkgs-unstable;
```

Comment:
Would it be possible to consolidate the example to use only a single (remote) `nixpkgs`?

Reply:
Sure, good idea - I'll do that